network/wsNetwork.go

```diff
@@ -730,7 +730,7 @@ func (wn *WebsocketNetwork) setup() {
 	)
 
 	wn.broadcastQueueHighPrio = make(chan broadcastRequest, wn.outgoingMessagesBufferSize)
-	wn.broadcastQueueBulk = make(chan broadcastRequest, 100)
+	wn.broadcastQueueBulk = make(chan broadcastRequest, 10000)
```

Comment:
```suggestion
	wn.broadcastQueueBulk = make(chan broadcastRequest, 10)
```

oops, shouldn't have sent upstream